pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -171,8 +171,8 @@ func TestNewYamlConfig(t *testing.T) {
 			args: args{
 				host:           host,
 				logoutRedirect: "",
-				brand:          "okd",
-				docURL:         "https://docs.okd.io/4.0/",
+				brand:          DEFAULT_BRAND,
```

Comment:
I wonder if we can do something like:

```golang
// configmap.go
func defaultBrand() string {
    return DEFAULT_BRAND
}
func defaultDocURL() sting {
   return DEFAULT_DOC_URL
}
// configmap_test.go
TestDefaultBrand () {} 
TestDefaultDocURL() {}
```

Reply:
Would that then require:

```golang
func getBrand(operatorConfig *operatorv1.Console) operatorv1.Brand {
	if len(operatorConfig.Spec.Customization.Brand) > 0 {
		return operatorConfig.Spec.Customization.Brand
	}
	return defaultBrand()  // because we can test a func, cant test a string
}
```

Reply:
It does add an additional wrapper `func`, but the testing would be fairly simple...

Reply:
Well, we could add these functions and do super simple unit test and if there is a problem (and we execute them first) then we might see there is an issue.  However, the other unit tests include the brand and doc link already as part of the deep copy so we would be testing it twice.  I know the output is not perfect, but perhaps we should work on better functionality to show the differences then write other helper functions.

Reply:
works for me.  Looking @ #159 as a possible solution.